### PR TITLE
Add linting to project

### LINKED ISSUE #19: Use same lint rules as yt-channel-info
Following the same code style in FreeTube modules will make maintaining a lot easier. This will require a large part of the code to be changed so I would like to have #18 and #17 merged before making this change, it will also introduce breaking changes (as method names will have to change)

### PR DESCRIPTION
**Pull request by ChunkyProgrammer**
- lint project (make some methods no longer public)
- add lint workflow
- update package version (method names changed)

It should now be more similar to https://github.com/FreeTubeApp/yt-comment-scraper and https://github.com/FreeTubeApp/yt-hashtag-scraper in structure

closes #19

### CODE REVIEW

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]

**Comment by ChunkyProgrammer** [comment]
Note: there is some code that gets reused between modules (ex: short video length extraction) and I think making a helper module would be useful so we don't have to duplicate code across modules. It could also help with fixing an issue that this will cause for FreeTube (A function from this module is used (but shouldn't be): https://github.com/FreeTubeApp/FreeTube/blob/3321fa91e42154d804650e67cd150c851b4ec0a0/src/renderer/views/Search/Search.js#L157)

File: src/HtmlParser.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      this.createThumbnailDictionary('end', `https://i.ytimg.com/vi/${videoId}/3.jpg`, 120, 90),
+    ]
+  }
+
+  static createThumbnailDictionary(Quality, Url, Width, Height) {
+    return {
+      quality: Quality,
+      url: Url,
+      width: Width,
+      height: Height
```
Comment (absidue) [inline src/HtmlParser.js:166]: As these aren't class names, it's probably a good idea to make these lowercase.
```suggestion
  static createThumbnailDictionary(quality, url, width, height) {
    return {
      quality: quality,
      url: url,
      width: width,
      height: height
```

**Review by absidue — COMMENTED** [review]
Just that minor thing, everything else looks good to me.

**Review by absidue — APPROVED** [review]

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@PikachuEXE 😬

**Review by PikachuEXE — APPROVED** [review]
Different time zone as always

**Pull request merged**